### ISSUE TITLE
Allow localisable "apply filters" text for the filter component

### ISSUE THREAD

**Issue by pandu-supriyono** [ministryofjustice/moj-frontend#486]
## Summary

Allow the hard-coded "apply filters" text in the filter component to be customisable.

## Motivation

We came across this requirement during the development of our app which is not in English. Related: [WCAG SC 3.1.2 Language of Parts](https://www.w3.org/WAI/WCAG21/Understanding/language-of-parts.html)

## Describe alternatives you've considered

For now, we've applied a patch (without thinking too much about the name of the parameter used to customise the text):

```diff
diff --git a/node_modules/@ministryofjustice/frontend/moj/components/filter/template.njk b/node_modules/@ministryofjustice/frontend/moj/components/filter/template.njk
index 5ebe066..26b944f 100644
--- a/node_modules/@ministryofjustice/frontend/moj/components/filter/template.njk
+++ b/node_modules/@ministryofjustice/frontend/moj/components/filter/template.njk 
@@ -48,7 +48,8 @@
     <div class="moj-filter__options">
 
       {{ govukButton({
-        text: 'Apply filters'
+        text: params.applyFilters.text or "Apply filters"
       }) }}
 
       {{params.optionsHtml | safe}}
```

A similar issue and use case is described in issue #404.

**Comment by pandu-supriyono** [comment]
Almost related: #492

**Comment by stalwartsolutions** [comment]
I've made this change, or something close to it at least (params.submit.text), as I was making related additions for my own project. :) It's been released in v1.8.0.
